UnicornEngine/sources/video/Renderer.cpp

```diff
@@ -133,77 +123,77 @@ void Renderer::Deinit()
         DestroyDebugReportCallbackEXT();
     }
 
-    for (VkImageView& view : m_swapChainImageViews)
+    for (vk::ImageView& view : m_swapChainImageViews)
```

Comment:
Some sort of. For example, logical device must be freed after m_vkSwapChain, because logical device destroy it and if logical device will be destroyed first, swapchain will be unfreed.